test/clearingHouse/ClearingHouse.badDebt.test.ts

```diff
@@ -56,30 +61,44 @@ describe("ClearingHouse badDebt", () => {
         await collateral.mint(bob.address, parseUnits("100", decimals))
         await deposit(bob, vault, 100, collateral)
 
+        // prepare collateral for carol
+        await collateral.mint(carol.address, parseUnits("1000000", decimals))
+        await deposit(carol, vault, 1000000, collateral)
+
         // alice add liquidity
         await addOrder(fixture, alice, "500", "50000", lowerTick, upperTick, false)
+
+        // initiate both the real and mocked timestamps to enable hard-coded funding related numbers
+        // NOTE: Should be the last step in beforeEach
+        await initiateBothTimestamps(clearingHouse)
     })
 
     describe("close/reduce position when bad debt", () => {
-        describe("taker has long position and market price becomes lower than index price", async () => {
+        describe("taker has long position and market price becomes lower than mark price", async () => {
             beforeEach(async () => {
                 // bob long base Token with 8x leverage
                 // bob open notional: -800
-                // bob position size: 7.866
+                // bob position size: 7.866265610482054835
                 await q2bExactInput(fixture, bob, "800", baseToken.address)
-                // market price = index price = 103.222
+
+                // To ignore founding payment
```

Comment:
```suggestion
                // To ignore funding payment
```

Reply:
fixed in 558a8d2db7c402d809df779c2a463cd5def4c129